index.html

```diff
@@ -4,23 +4,16 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
```

Comment:
- [ ] Please return this `<link>` used for your hamburger icon. The icon disappears
<img width="715" alt="Screen Shot 2022-08-20 at 10 01 42 PM" src="https://user-images.githubusercontent.com/47673311/185771139-588fa464-f050-4e13-8e64-4663b97f9eb4.png">
 after the last review.

Reply:
ok i do it